src/pages/search/desktop/search.module.scss

```diff
@@ -1,19 +1,28 @@
 .container {
-  height: calc(100vh - var(--desktop-nav-height));
-  overflow-y: scroll;
+  @media screen and (min-width: 600px) {
```

Comment:
we should use tailwind styles instead of media query

Reply:
At the time of implementing this there was no tailwind :)
I shall refactor asap